manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -375,6 +381,8 @@ jobs:
             uaa: {}
       metron_agent:
         zone: z1
+      route_registrar:
+        routes: []
```

Comment:
Ah, thanks for that link.  Given it's fixed upstream, can we add a `FIXME` comment so that we know to remove this when we upgrade.